web-component/package.json

```diff
@@ -39,8 +39,7 @@
   "scripts": {
     "start": "tsc --watch",
     "bundle": "webpack",
-    "prepare": "npm run bundle",
```

Comment:
I'll bring it back, it is still needed in https://github.com/asyncapi/asyncapi-react/blob/master/package.json#L21

will commit directly to your branch to not bother you with tiny stuff